ev3dev2/__init__.py

```diff
@@ -242,20 +252,22 @@ def _raise_friendly_access_error(self, driver_error, attribute):
         if not isinstance(driver_error, OSError):
             raise driver_error
 
-        if driver_error.errno == errno.EINVAL:
+        driver_errorno = driver_error.args[0] if is_micropython() else driver_error.errno
```

Comment:
It would feel nicer if this would be implemented as a function (`driver_errno()`) but I guess it is OK as is, since this is the only place we need it.